docs/root/configuration/http_filters/rate_limit_filter.rst

```diff
@@ -5,6 +5,7 @@ Rate limit
 
 * Global rate limiting :ref:`architecture overview <arch_overview_rate_limit>`
 * :ref:`v2 API reference <envoy_api_msg_config.filter.http.rate_limit.v2.RateLimit>`
+* This filter should be configured with the name *envoy.rate_limit*.
```

Comment:
@mattklein123  I could not find any better place other than here to document these names. LMK if you think there is any better place.

Reply:
It's already documented in https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/listener/listener.proto and https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/filter/network/http_connection_manager/v2/http_connection_manager.proto#envoy-api-msg-config-filter-network-http-connection-manager-v2-httpfilter, but I think it makes sense to put the documentation of the filter names here also (optimally there would be a config snippet also).

If we do this, can you do something similar for all the other filters so we have some consistency?

Reply:
Yeah. I saw that. But I felt it is hidden/not obvious. Having it here would make it more clear. I will add for rest of the filters.

Reply:
@mattKlein123 Added filter names for all filter just in the configuration section. On the config part, I did not do because it may be too much of maintenance to keep it in sync with the changes. Any way, we are giving links to config section just below here. Also since most of the filter configs are generated by management server, not sure if it helps to give a snippet here. LMK, if you think it would be useful, I will add.